sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/SymbolicExecution/ExplodedGraph.cs

```diff
@@ -851,6 +851,12 @@ protected override bool IsValueConsumingStatement(SyntaxNode jumpNode)
                 return returnStatement.Expression != null;
             }
 
+            var switchStatement = jumpNode as SwitchStatementSyntax;

```

Comment:
This method determines whether a value should be popped from the stack. Now we are handling both switch and try/catch in VisitBranchBlock in Common.ExplodedGraph, hence we need a way to determine whether to pop (switch) or not to pop (try) from stack...